code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -49,13 +49,13 @@
 		if(L.getStaminaLoss() >= 60)
 			L.Sleeping(300)
 		else
-			L.adjustStaminaLoss(25) 
+			L.adjustStaminaLoss(25)
 
 // .32 ACP (Improvised Pistol)
 
 /obj/item/projectile/bullet/c32acp
 	name = ".32 bullet"
-	damage = 13
+	damage = 15	//Skyrat Edit: 13 => 15
```

Comment:
Could probably get away with buffing to 17 damage, since the weapon is very very slow.

Reply:
It's faster firing now, and I need the round to be 15 for other stuff in the future.